service.go

```diff
@@ -361,7 +364,24 @@ func (s *Service) LoadDNA(path string, filename string, format string) (dnaP *DN
 		if zome.BridgeTo != "" {
 			dna.Zomes[i].BridgeTo, err = NewHash(zome.BridgeTo)
 			if err != nil {
-				return
+				// if in dev mode assume the bridgeTo was the app name
+				// and that hcdev put the actual DNA for us in the env var
+				// so try again with that value
+				if IsDevMode {
+					dnaHashStr := os.Getenv("HCDEV_DNA_FOR_" + zome.BridgeTo)
```

Comment:
You wanted to change this to a global variable, @zippy, right?

Reply:
arg, I did this but didn't commit it..  I will

Reply:
Oh, I figured it it, it was commited as part of  369, and this branch is just behind.  I will merge master into it.